projects/systelab-translate/package.json

```diff
@@ -19,8 +19,8 @@
   },
   "private": false,
   "peerDependencies": {
-    "@angular/common": "^16.0.0",
-    "@angular/core": "^16.0.0",
+    "@angular/common": "^17.0.0",
+    "@angular/core": "^17.0.0",
     "@ngx-translate/core": "^15.0.0",
     "@ngx-translate/http-loader": "^8.0.0",
     "date-fns": "^2.30.0"
```

Comment:
We should consider to upgrade date-fns to version 3.x. Last version is 3.20, so we should be able to use the latest one. Let's evaluate impact and if it's too much, let's add a new ticket to do it separatedly (i.e maybe as part of next PI upgrade),

Reply:
I tried to update it but there are a lot of changes to be made and a lot of errors regarding that. I'll be creating a new issue to update it